content/en/references/network-troubleshooting/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Network troubleshooting
+weight: 2
+tags:
+- troubleshooting
+- networking
+description: >
+  How to troubleshoot common network problems
+---
+
+## Overview
+
+Below are several scenarios in which you may be trying to use LocalStack.
+If you are having difficulties connecting your application code to LocalStack, please visit the links below each section which go into further details.
+
+{{<alert title="Note">}}
+LocalStack only binds to IPv4 addresses (e.g. `127.0.0.1`). Check you are not trying to access LocalStack over IPv6.
+{{</alert>}}
+
+---
+
+## Accessing LocalStack directly
+
+{{< figure src="./images/overview-1.svg" width="400" >}}
+
+For example with the [AWS CLI]({{< ref "user-guide/integrations/aws-cli" >}}) where you need to specify the URL of LocalStack yourself, such as:
+
+{{< command >}}
+aws --endpoint-url http://localhost.localstack.cloud:4566 <command>
+# or
```

Comment:
Thanks for pointing me to the

> Across our docs, we use localhost:4566 instead of localhost.localstack.cloud

section, that's good to know. I'll update the PR with this instead. I think if I remove the reference to `localhost.localstack.cloud` then I don't need to additionally link to the rebind protection here, but I do mention "any domain name that refers to `locahost`" in another page, and I'll link the rebind protection doc there. Thanks!